### ISSUE TITLE
Value generation from specs

### ISSUE THREAD

**Issue by rchanou** [prayerslayer/js.spec#14]
This is one of the more interesting features of clojure.spec. Are methods like [exercise ](https://clojure.github.io/clojure/branch-master/clojure.spec-api.html#clojure.spec/exercise)on the roadmap?

On a related note, what are your thoughts on[ testcheck-js](https://github.com/leebyron/testcheck-js) and how it compares to js.spec?

**Comment by prayerslayer** [comment]
Yes, it's definitely a cool feature of clojure.spec. I would like to have it in js.spec as well, by using testcheck-js if possible. The "roadmap" is basically [this part](https://github.com/prayerslayer/js.spec#implementation-status) of the README, so once I have the regex support finished I would look into generators.

**Comment by rchanou** [comment]
Oh, I saw that but for some reason thought that referred to `function*` ES2015 generators 🤦😅 Sounds good!
